packages/kbn-grid-layout/grid/grid_height_smoother.tsx

```diff
@@ -22,8 +26,26 @@ export const GridHeightSmoother = ({
     const subscription = combineLatest([
       gridLayoutStateManager.gridDimensions$,
       gridLayoutStateManager.interactionEvent$,
-    ]).subscribe(([dimensions, interactionEvent]) => {
+      gridLayoutStateManager.expandedPanelId$,
```

Comment:
I think this can also be a seperate subscription? We only need to fire this when `expandedPanelId$` emits, right?

Reply:
I'm also starting to wonder if this could be controlled via inline styles and/or class names. 

The reason we control the panel styles during dragging via a subscription was to avoid the **many many** re-renderings that were happening by relying on props to do this. But I think we might be able to do something similar to what you did for disabling the drag + resize handles? 🤔